### PR TITLE
add missing snippet for protocols

### PR DESCRIPTION
**Pull request by dayhaysoos**
Adding missing code snippet to the docs site

### CODE REVIEW

**Review by angiejones — APPROVED** [review]

File: site/code-snippets/api/web5-js/dwn/protocols.js
Comment (angiejones) [inline site/code-snippets/api/web5-js/dwn/protocols.js:1]: heads up, I've updated this code on the site so this needs to be updated

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/code-snippets/api/web5-js/dwn/protocols.js
Comment (dayhaysoos) [reply site/code-snippets/api/web5-js/dwn/protocols.js:1]: Taking a look now

Comment (dayhaysoos) [reply site/code-snippets/api/web5-js/dwn/protocols.js:1]: Hey Angie, I'm looking at the code snippet now and it looks exactly the same as the one I replaced:

```
const { protocols } = await web5.dwn.protocols.query({
  from: "did:example:bob",
  message: {
    filter: {
      protocol: "https://music.org/protocol",
    },
  },
});

// logs an array of protocol configurations installed on Bob's DWN
console.log(protocols);
```

Is this the one you were referring to?

Comment (angiejones) [reply site/code-snippets/api/web5-js/dwn/protocols.js:1]: sorry! i changed the code on the Protocol tutorial, not the API reference. Sorry for the confusion. No action needed

**Review by angiejones — APPROVED** [review]

**Pull request merged**